guides/configuration.md

```diff
@@ -112,6 +112,7 @@ Environment variables to configure a production release.
 
 #### General settings:
 * `PUSH_LOGLEVEL` - `debug`/`info`/`warn`/`error` - Log level of the application. `info` is the default one
+* `PUSH_LOGFORMAT` - `logfmt`/`json` - Log format of the application. Defaults to `logfmt` for `dev` and `test` environments and to `json` for `prod` environment.
```

Comment:
```suggestion
* `PUSH_LOGFORMAT` - `logfmt`/`json` - Log format of the application. Defaults to `logfmt` for the `dev` and `test` environments, and to `json` for the `prod` environment.
```
As we're mentioning specific environments, it needs a definite article. Wouldn't need one if it was an indefinite plural, like "for development environments", and not "for the development environments", but in this case the envs have specific names 🤷‍♂️